### PR TITLE
Adds 2 new moth plushies

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

Adds 2 new moth plushies

## Why It's Good For The Game

Every moth has a representation in the form of a plushie. Now the witchwing has it too! The medical moff is a silly looking plushie added to the game for the sake of fun. It's also something that will be added as an easter egg in the Kilo Medbay  map rework, hidden in maintenance.
## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/9b4d189f-53f0-475d-8131-33882c449563

imagen
imagen

</details>

## Changelog
:cl: Varo
add: New moth plushie: Witch Wing
add: New moth plushie: Medical Moff
/:cl:

### CODE REVIEW

**Comment by Rukofamicom** [comment]
Please properly fill out the entire body. 

What are these plushes based on and why should they be included in the game? Are they good for the game and how do they improve it?

**Comment by ClownMoff** [comment]
Done

**Review by Rukofamicom — APPROVED** [review]
The simple code is fine, waiting for sprite review by Pigeon. 

I'd suggest posting the sprites directly within the body to make reviewing them easier.

**Pull request merged**
